server/pkg/thread/comment.go

```diff
@@ -27,3 +37,15 @@ func (c *Comment) CreatedAt() time.Time {
 func (c *Comment) SetContent(content string) {
 	c.content = content
 }
+
+func (c *Comment) Clone() *Comment {
+	if c == nil {
+		return nil
+	}
+
+	return &Comment{
+		id:      c.id.Clone(),
+		author:  c.author.Clone(),
```

Comment:
```suggestion
		id:      c.id,
		author:  c.author,
```

Clone is not needed because they are not references

Reply:
Done!